frontend/amp/components/elements/RichLinkBlockComponent.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { css } from 'emotion';
+import { palette } from '@guardian/pasteup/palette';
+import { serif, sans } from '@guardian/pasteup/fonts';
+import { pillarPalette } from '@frontend/lib/pillars';
+
+export const RichLinkBlockComponent: React.SFC<{
+    element: RichLinkBlockElement;
+    pillar: Pillar;
+}> = ({ element, pillar }) => {
+    if (element.sponsorship) {
+        throw new Error('Sponsored rich links not supported');
+    }
+    return (
+        <aside className={style}>
+            <a className={linkStyle(pillar)} href={element.url}>
+                {element.text}
+            </a>
+        </aside>
+    );
+};
+
+const style = css`
+    float: left;
+    width: 130px;
+    padding: 4px;
+    padding-bottom: 18px;
+
+    margin: 4px 10px 12px 0;
+    background-color: ${palette.neutral[93]};
+    border-top: 1px solid ${palette.neutral[86]};
+    margin-right: 20px;
+`;
+const linkStyle = (pillar: Pillar) => css`
+    color: ${pillarPalette[pillar].main};
+    font-family: ${serif.body};
+    font-weight: 700;
+    border: 0;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    word-wrap: break-word;
+    :hover {
```

Comment:
More as a comment, in some sense we don't really need to support hover on AMP as most (nearly all?) mobile devices don't care about hover.

But we've added a few elsewhere too mostly because it is nicer when you are testing things on desktop.

So I don't really mind, but certainly don't feel obligated to put them in!

Reply:
honestly was just going for maximum parity with existing styles.

Reply:
ha yeah. In a way it is quite nice to have at least some interactivity when testing on desktop.